scripts/run_portserver_test.py

```diff
@@ -381,12 +381,25 @@ def dummy_handler(data: bytes) -> bytes:
 
         with swap_socket, swap_hasattr:
             server = run_portserver.Server(dummy_handler, '\08181')
-            run_portserver.Server.handle_connection(MockSocket(), dummy_handler)
-
-            self.assertFalse(server.socket.server_closed)
+            # Here we use MyPy ignore because handle_connection method expects
```

Comment:
We should avoid introducing new ignores (both here and everywhere below). Why can't we construct MockSocket so that it has the correct type?